### PR TITLE
undo quorum=true; wait for unit creation in fleetctl

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

File: fleetctl/fleetctl.go [hunk 1]
```diff
@@ -660,7 +662,26 @@ func lazyCreateUnits(args []string) error {
...
+	}
+
+	go func() {
+		wg.Wait()
+		close(errchan)
+	}()
+
+	haserr := false
+	for msg := range errchan {
+		stderr("Error creating units: %v", msg)
```
Comment (marineam) [inline fleetctl/fleetctl.go]: "error waiting on unit creation" would be more accurate.

File: fleetctl/fleetctl.go [hunk 2]
```diff
@@ -660,7 +662,26 @@ func lazyCreateUnits(args []string) error {
...
+		haserr = true
 	}
+
+	if haserr {
```
Comment (jonboulle) [inline fleetctl/fleetctl.go]: maybe https://github.com/coreos/pkg/pull/33 ?

**Comment by mischief** [comment]
@jonboulle looks useful, but not merged yet :(

**Comment by mischief** [comment]
PTAL

**Comment by mischief** [comment]
@marineam ping

**Comment by marineam** [comment]
LGTM

**Comment by jonboulle** [comment]
:+1:

**Comment by philips** [comment]
@mischief multierror is merged.

**Comment by hawka** [comment]
Can you explain why this was chosen over the use of quorum=true? What are the benefits to this approach?

**Comment by mischief** [comment]
we maintain compatibility with etcd 0.4.x for now.

**Comment by marineam** [comment]
iirc quorum=true wasn't a 0.4 limitation, it would just be overly expensive for a large fleet cluster to always use quorum=true for everything.

**Pull request merged**